src/radio-button.tsx

```diff
@@ -144,8 +144,11 @@ const RadioButton: React.FC<PropsRender | PropsChildren> = ({value, id, dataAttr
             {rest.render ? (
                 rest.render(radio, disabled)
             ) : (
-                <Inline space={16} alignItems="center">
-                    {radio}
+                <Inline space={16}>
```

Comment:
There is a little extra pixel in radio button that make the text is displaced to the right?
![image](https://user-images.githubusercontent.com/6722153/145577449-56164d7c-d7cc-4c8b-b0be-204d35c1dcfa.png)

https://mistica-web-oo1gboweb-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AhCAPABABwIZSgEsA7AcwF5gBGANgF8A%2BAHWM0yQGUAXXMAa0wBnPGBiUATABZGLNkgDCACxj8ARhkzFcAWzFNwytRn3NWbTABkIAJxjbMhbIICu9qBAA2NoYS6YdMFwANJiQxIIqXIHO1v5E2ISCYCSkmDAeviERUJjuaYTOgtrQsuZR2tjeJMlEUM7Efs5%2BHrjqtml%2B7l7t2rikWv4ZAI7OuAB0mACqfjDEhPb4mNok86VsAG6zhDohI4maEIJc1s45MOgw1sk8XIQQrM4eLdqQmC1t%2B3OChJhNg4R7bBpLAwXBrUIQbTFPJhCIjXBcMalJAAeiUKj46nQpjkACV8HcAOLWCDObDgrS6cj6ABWzkIYAAngBaABmJ18%2BnBuGs22Z73SHhgUFUjOpIAFHi5ZjYsFZuEeXAAargPM49CBVLgtFppWwceZODx%2BEIRGJqPQDeZ2PiiBAUE0uPdMOtVerxVqdWCQFbrZYbHYHE5XLlPN5vn4AsEIeFItFYgSEkkUmkMtHsqHwdaYAUitBMOVKrFqoRavVGs1WgGOqHujAln0BqqAaMJtM0nMFlAs%2BZlp2XVsdpg9oIDkcTmcLldfAi7g8njpXu8bJ9Ej8mj22M3AcC0rgIVDoBAY3DRoie6jbXcHVwncRfajuLw%2BAbLwSIMTSdgHyin-xTKiaDYiwIB0EAA

Reply:
checkbox size is 18px width, radio button is 20px width, this was this way before these changes, but I could change this, what is the correct value, 18 or 20?

Reply:
both is correct but in design we have both of thems inside a box with size 24x24

Reply:
understood, @pladaria  @atabel did you have these kind of issue in the past? I could add a wrapper to the icons of 24px width, WDYT?

Reply:
![image](https://user-images.githubusercontent.com/579417/145789948-be32bef7-999b-47d2-9a9a-e49a23906fbb.png)
What @yceballost comments is not documented in radio button/checkbox specs

If you want to align the text simply put a smaller spacing (between the radio and the text) in radio buttons

Reply:
It's true, is not specified, but you can select any control in spec and see this bounding box. So we can remove it from design or add it to code. 

why a 24px wrapper could break implementations?

Reply:
Not break but it will probably displace the content in unexpected ways